### ISSUE TITLE
Missing size data for New Blazor Template - Publish - AOT

### ISSUE THREAD

**Issue by SamMonoRT** [dotnet/performance#1832]
From https://msit.powerbi.com/groups/me/apps/54e0e83f-07bc-45bf-87b7-a7677ff3af2a/dashboards/fa051820-ff60-4d40-8a08-bdcc1b47b1d0

The last entry for the New Blazor Template - Publish - AOT app is from May 12th. @DrewScoggins is this a known issue and related to https://github.com/dotnet/performance/issues/1820 or something different.

![image](https://user-images.githubusercontent.com/46026722/120547985-ccf96b80-c3bf-11eb-95cd-92e631fc8942.png)

**Comment by DrewScoggins** [comment]
AOT has been blocked by this issue, https://github.com/dotnet/sdk/pull/17794#issuecomment-852222860. The fix went into the release branch, but not main. Once this fix goes in we should have these runs back up.

**Comment by DrewScoggins** [comment]
There was another issue that was blocking us here. The steps for getting the latest build of the SDK was changed, I have now added code on our end to mitigate this change, and we are getting AOT results. So I am closing this issue.
